moveit_ros/planning/kdl_kinematics_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(MOVEIT_LIB_NAME moveit_kdl_kinematics_plugin)
```

Comment:
hey @davetcoleman  Dave thanks for reviewing! yep you are right, there is noting to do with KDL plugin.  I forgot why i added it.  Tones of moveIt stuff I don't know. Is kdl also deprecated or is self-contain in newest version? maybe be the habit from gazebo.

We are try moveIt! on a new collaborative arm robot  **techman robot** made from taiwan
http://tm-robot.com/
Now just made rviz could connect to real robot and gazebo model;  And there is some challenge in motion plannning like RRTconnectkConfig; some poses the robot could not reach or the plannig path it's strang. Is there some reference for add constrain or specified the motion?  
here are repo's 
my lab: https://github.com/ISCI-NCTU/isci_ros_tm5
official: https://github.com/kentsai0319/techman_robot
Just in case you guys get interest and have free time to play, hehehe.  
Anyway, this is not relate to this PR.  Thanks for review! :smile: 

![moveit_configuration](https://cloud.githubusercontent.com/assets/7314531/24085448/50d2ef56-0d37-11e7-97d9-e9d40e8cdca6.png)